identity-account/src/account/account.rs

```diff
@@ -176,6 +176,10 @@ where
 
   /// Returns the DID document of the identity, which this account manages,
   /// with all updates applied.
+  ///
+  /// Note that the returned document only has a valid signature after publication.
+  /// In general for use cases where the signature is required, it is advisable to resolve the
+  /// document from tangle, for instance by using [`Account::resolve_identity`].
```

Comment:
Comment. This is technically untrue since publishing with a diff update (which are still not removed so we have to assume they'll come back at some point) doesn't sign the document.

Function name casing is different in Wasm, removed for convenience.

```suggestion
  ///
  /// Note: the returned document only has a valid signature after publishing an integration chain update.
  /// In general, for use cases where the signature is required, it is advisable to resolve the
  /// document from the Tangle.
```